docs/usage/getting-started/running.md

```diff
@@ -1,70 +1,82 @@
 
-If someone else is hosting Renovate for you, or you are using the Mend Renovate App on GitHub, then you can skip ahead to the [installing & onboarding](./installing-onboarding.md) page.
+If you're using the Mend Renovate App on GitHub, or if someone else is hosting Renovate for you, skip ahead to the [installing & onboarding](./installing-onboarding.md) page.
 
 ## Self-Hosting Renovate
 
-Self-hosting Renovate means that you are the "administrator" of the bot, which entails:
+When self-hosting Renovate you're the "administrator" of the bot, this means you:
 
-- You provide infrastructure for it to run on,
-- You provision its global config,
-- You ensure it's running regularly,
-- You ensure Renovate bot itself is updated
+- provide the infrastructure that Renovate runs on,
+- provision Renovate's global config,
+- make sure Renovate bot runs regularly,
+- make sure Renovate bot itself is updated
 
-Read our section on [Self-hosting on Windows](./installing-onboarding.md#self-hosting-on-windows) to prevent line endings from confusing Renovate bot.
+If you're self-hosting Renovate on Windows, read [Self-hosting on Windows](./installing-onboarding.md#self-hosting-on-windows) to prevent line endings from confusing Renovate bot.
 
 ### Available distributions
 
 #### npm package (CLI)
 
 Renovate's Open Source CLI is built and distributed as the npm package `renovate`.
-You can run this directly in any Node.js environment - even via `npx` - and it will process all the repositories it is configured with, before exiting.
-When you install Renovate from npm it naturally does not come bundled with any third-party tools or languages such as Ruby, Python, Composer, Bundler, Poetry, etc.
+You can run this package in any Node.js environment - even via `npx` - and it will process all the repositories it is configured with, before exiting.
+
+Renovate's npm package does not bundle any third-party tools or languages like Ruby, Python, Composer, Bundler, Poetry, etc.
 If you need Renovate to support any non-npm lock files like Bundler then you'll need to make sure all required third-party tools are pre-installed in the same environment alongside Renovate before you run it.
 
 The `renovate` npm package is compatible with all of Renovate's supported platforms.
 
-Renovate requires Node.js `>=14.15.0` and Git `>=2.33.0`.
+Renovate requires Node.js `>=18.12.0` and Git `>=2.33.0`.
 
 #### Docker image
 
-The `renovate` npm package is also distributed via pre-built Node.js images on Docker Hub (`renovate/renovate`).
+The `renovate` npm package is also distributed via Node.js images on Docker Hub (`renovate/renovate`).
+These Docker images work on all the platforms that Renovate supports.
+
+There are two flavors:
+
+- The "full" image (the `latest` tag defaults to the full image)
+- The `-slim` image
+
+##### The full image
 
-The `slim` image contains only Node.js so works if either:
+The "full" image comes with every package manager that Renovate supports.
```

Comment:
Docker Hub has a `-slim` tag for the "slim" images, but I can't find a `-full` tag for the "full" images. Is that intended?

I think we mean that the full/default image has all the stuff included. But it's a bit confusing to call it "full" when there's no `-full` tag on Docker Hub.

What do you think?

Reply:
Yes, if there's no suffix then it means it's full. Maybe we can say the default image